bin/wget.py

```diff
@@ -24,17 +29,20 @@ def main(args):
     console.show_activity()
     try:
 
-        print 'Opening: %s' % url
+        sys.stdout.write('Opening: %s\n' % url)
```

Comment:
I think all of these `sys.stdout.write` calls could be translated to `print` calls.